deal.II-toolchain/packages/boost.package

```diff
@@ -1,4 +1,5 @@
-VERSION=1_60_0;CHECKSUM=28f58b9a33469388302110562bdf6188
+VERSION=1_63_0;CHECKSUM=7b493c08bc9557bbde7e29091f28b605
+#VERSION=1_60_0;CHECKSUM=28f58b9a33469388302110562bdf6188
```

Comment:
The patch is already merged, but in general there is no sense leaving the old text as a comment.